.github/ISSUE_TEMPLATE/001_bug_report.yml

```diff
@@ -0,0 +1,83 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+type: "Bug"
+projects: ["octo-org/1", "octo-org/44"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+
+      placeholder: |
+        Please also mention any information which could help others to understand
+        the problem you're facing:
+          - What target platform are you using?
+          - What have you tried to diagnose or workaround this issue?
+          - Is this a regression? If yes, have you been able to "git bisect" it to a
+            specific commit?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Steps to reproduce the behavior.
+
+      placeholder: |
+        Steps to reproduce the behavior:
+          1. mkdir build; cd build
+          2. cmake -DBOARD=board\_xyz
+          3. make
+          4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: dropdown
+    attributes:
+      label: Impact
+      description: Impact of this bug
```

Comment:
Sad to lose the freetext here, sometimes there's good information in it.

Reply:
there is enough free text fields to add this type of information.